enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -1984,13 +1984,13 @@ func (cdcw *cdcWriter) writeRawChunk(fileRecord *rfpb.FileRecord, key filestore.
 	}
 	_, err = wcm.Write(chunkData)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to write raw chunk: %s", err)
```

Comment:
any reason not to use status.Blah() everywhere?

Reply:
Done.